### PR TITLE
fix: Adds bug fixes for menu and font size.

### PR DESCRIPTION
**Pull request by mikaelbr**
- Should fix issue with menu being off when navigating back in Safari.
- Should also add better keyboard navigation when navigation is visible and scrollable area
- And lastly, reduces max size of font by upgrading @varianer/profile dependency where it has been fixed permanently.

### CODE REVIEW

File: src/layout/index.tsx [hunk 1]
```diff
@@ -17,43 +17,19 @@ const Layout: React.FC<LayoutProps> = ({
...
-      return -1;
-    }
-  }
+  const modalRef = React.createRef<HTMLDivElement>();
+  const navRef = React.createRef<HTMLUListElement>();
+  const {
+    isMenuVisible,
+    setMenuVisible,
+    tabIndex,
+    // isNotHamburgerMode,
```
Comment (adriansberg) [inline src/layout/index.tsx:26]: Remove?

File: src/layout/index.tsx [hunk 2]
```diff
@@ -269,3 +248,92 @@ const Layout: React.FC<LayoutProps> = ({
...
+        return e.preventDefault();
+      }
+
+      // On "reversed" tabbing. If previous element is outside modal, jump to last element
+      if (e.shiftKey && !modalRef.current?.contains(previous)) {
+        last?.focus();
+        return e.preventDefault();
+      }
+
+      // Normal tab flow.
```
Comment (adriansberg) [inline src/layout/index.tsx:314]: Misplaced comment or just me not understanding?

Comment (mikaelbr) [reply src/layout/index.tsx:314]: Can specify better! Point is, no case hits, so just follow normal browser behaviour.

Comment (adriansberg) [reply src/layout/index.tsx:314]: Right, gotcha.

**Comment by adriansberg** [comment]
I can't close the menu with the cross 😳

**Comment by mikaelbr** [comment]
Good catch! Fixed now

**Review by adriansberg — APPROVED** [review]

**Pull request merged**